### ISSUE TITLE
Add "Details" button to Track menu

### ISSUE THREAD

**Issue by Velomeloman** [osmandapp/OsmAnd#16484]
# 🚀 feature request

"Navigation" screen contains "Details" button, that show information about Road type, Surface, etc. 
![Screenshot_20230217_142147_net.osmand.plus.jpg](https://user-images.githubusercontent.com/109347561/219639939-a1050db0-a158-4015-8496-38497a4c2593.jpg)

Would be great to add same button to the Track screen.

![IMG_20230217_142353_903.jpg](https://user-images.githubusercontent.com/109347561/219640679-c9b9719e-9d38-49f3-9310-00b14d1f6aee.jpg)

**Comment by Zirochkabila** [comment]
@Velomeloman  The Details button works when a route is built. Accordingly, if the route is not built, there is no data.

**Comment by sonora** [comment]
That is true: The GPX file alone is mostly an assortment of location coordinates with limited or often no extra information contained. Use the GPX to build a route along it, then Details becomes available (by linking the GPX location info to map data like e.g. roads).

**Comment by Velomeloman** [comment]
Same track can be colored with any type of data. So not big deal to calculate it for tracks attached to roads by pressing Details. It happens immediately in edit mode and follow track mode.
If it not attached, just show a message to user with suggestion.
https://user-images.githubusercontent.com/109347561/219725802-0a660ade-2e81-4b33-8e20-90a22da75dc6.mp4

**Comment by punyidea** [comment]
> Same track can be colored with any type of data. So not big deal to calculate it for tracks attached to roads by pressing Details. 

Thanks for the tip! It seems that this feature works if the track itself is attached to roads, just like you mention( I tried this for a track on my phone and got a message that said the surface data was not in the track). In what is potentially a new feature, could there be an added enhancement that queries the road surfaces under the track for this information? It seems this feature may be valuable if a track is long and takes time to attach to roads, with potentially some added information that attaching roads may take time. (not sure if the process of querying all surfaces would take less time).

**Comment by vshcherb** [comment]
Important Note: this task is closely https://github.com/osmandapp/OsmAnd/issues/17937 (recorded track no HR graph)

**To design** 
1. Analyze on map could provide such information similar to Heart Rate, Plan 
2. On track screen we need to simplify graph and list views. Currently we display all graphs / short info (altitude, speed) for all segments, it takes a lot of UI space. So we need to display either 1) All Graphs for 1 general track 2) Have inner screen to view information about segment like in Navigation menu 

**Workaround**
- Plan route already provides road type info graphs
